src/ocm/components/AddHosts/HostsClusterDetailTab.tsx

```diff
@@ -127,21 +128,24 @@ const HostsClusterDetailTabContent: React.FC<HostsClusterDetailTabProps> = ({
           setDay2Cluster(day2Cluster);
         } catch (e) {
           handleApiError(e);
-          setError(
-            <>
-              {e.request._url.includes('/import')
-                ? 'Failed to create wrapping cluster for adding new hosts.'
-                : 'Failed to fetch cluster for adding new hosts.'}
-              <br />
-              Check your connection and <TryAgain />.
-            </>,
-          );
+          if (isApiError(e) && e.request) {
+            // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-call
```

Comment:
Could we remove the exceptions? They are already warnings from .estlintrc, and it makes them easier to find then when we remove the exception.